apps/editing-toolkit/editing-toolkit-plugin/help-center/class-help-center.php

```diff
@@ -127,7 +127,17 @@ public function enqueue_script() {
 	 * Get current site details.
 	 */
 	public function get_current_site() {
-		$site     = \Jetpack_Options::get_option( 'id' );
+		$is_support_site = defined( 'WPCOM_SUPPORT_BLOG_IDS' ) && $this->is_support_site();
```

Comment:
This `defined` check should be part of `is_support_site` since that's the function's job.

Reply:
Absolutely 👍